### PR TITLE
revisit the final part of OR only

### PR DESCRIPTION
**Pull request by wlammen**

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -7652,18 +7661,15 @@ an illustration of the conservativity of definitions (definitions do not
...
-    ( wi pm2.27 imori ) AABCBCABDE $.
-
   $( Theorem *2.42 of [WhiteheadRussell] p. 106.  (Contributed by NM,
      3-Jan-2005.) $)
   pm2.42 $p |- ( ( -. ph \/ ( ph -> ps ) ) -> ( ph -> ps ) ) $=
     ( wn wi pm2.21 id jaoi ) ACABDZHABEHFG $.
 
-  $( Theorem *5.11 of [WhiteheadRussell] p. 123.  (Contributed by NM,
-     3-Jan-2005.) $)
+$(  Implication and disjunction. $)
```
Comment (avekens) [inline set.mm:7669]: What does this comment mean at this place? There are other theorems before containing Implications and disjunctions.

File: set.mm [hunk 2]
```diff
@@ -7641,6 +7641,15 @@ an illustration of the conservativity of definitions (definitions do not
...
   $}
+  
+  $( Rewriting implication based theorems using disjunction. $)
+  
+  $( Theorem *2.26 of [WhiteheadRussell] p. 104.  See ~ pm2.27 .  (Contributed by NM,
+     3-Jan-2005.)  (Proof shortened by Wolf Lammen, 23-Nov-2012.) $)
+  pm2.26 $p |- ( -. ph \/ ( ( ph -> ps ) -> ps ) ) $=
+    ( wi pm2.27 imori ) AABCBCABDE $.
+
+  $( Contradiction and disjunction. $)
```
Comment (benjub) [inline set.mm:7652]: Do you mean "Negation and disjunction" ?

File: set.mm [hunk 3]
```diff
@@ -7652,17 +7661,14 @@ an illustration of the conservativity of definitions (definitions do not
...
-  pm2.26 $p |- ( -. ph \/ ( ( ph -> ps ) -> ps ) ) $=
-    ( wi pm2.27 imori ) AABCBCABDE $.
-
   $( Theorem *2.42 of [WhiteheadRussell] p. 106.  (Contributed by NM,
      3-Jan-2005.) $)
   pm2.42 $p |- ( ( -. ph \/ ( ph -> ps ) ) -> ( ph -> ps ) ) $=
     ( wn wi pm2.21 id jaoi ) ACABDZHABEHFG $.
 
-  $( Theorem *5.11 of [WhiteheadRussell] p. 123.  (Contributed by NM,
+$(  Implication and disjunction. $)
```
Comment (benjub) [inline set.mm:7669]: "implication" is always implied, so do you mean " Disjunction", or "Negation an disjunction" ?

Comment (wlammen) [reply set.mm:7669]: Some simple expressions connecting implication and disjunction.  I changed the title.

File: set.mm (hunk 2 above)
Comment (wlammen) [reply set.mm:7652]: No, the following theorems are pm2.65 in disguise.  And this expresses contradiction.

**Review by avekens — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (wlammen) [reply set.mm:7669]: At the end of each section are those theorems that belong formally to it, but provide nothing essential for the theory.  They are sort of applications of the foundation.  I tried to group them into similar theorems with some common idea, and the comment functions as a title.  I copied the idea from one of the sections that have been reworked (checkout the version of 6-Aug):

`$( Prove formula-building rules for the biconditional connective. $)`

was one of them.  I thought comments like this help maintaining a rather loose set of theorems.

**Review by benjub — APPROVED** [review]

**Pull request merged**